app/livechat/client/views/app/tabbar/customerChatHistoryMessages.js

```diff
@@ -0,0 +1,51 @@
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import './customerChatHistoryMessages.html';
+import { APIClient, t } from '../../../../../utils/client';
+import { Session } from 'meteor/session';
+
+let  Messages 
+let len, msgs
+let allHistoryChat ;
+
+Template.customerChatHistoryMessages.helpers({
+	historyResult() {
+		// will return all the messages in history room
+		return Template.instance().historyResult.get().reverse();
+    },
+    len() {
+		// will return length of total messages in room
+		len = Template.instance().historyResult.get();
+		return len = len.length-1;
+	},
+})
+Template.customerChatHistoryMessages.onCreated(function() {
+    this.historyResult = new ReactiveVar([]);
+    var id = Session.get('FetchID');
+    var token = Session.get('FetchToken')
+    this.autorun(async () => {
+		Messages  =  await APIClient.v1.get(`livechat/messages.history/${ id }?token=${token}`); 
+    // will return pervious chats list
+        let history = Messages.messages;
+        let newHisTory = []
```

Comment:
All of the code below is unnecessary.

Reply:
removed the unnecessary code only required code is here